scripts/enum/mod.lua

```diff
@@ -410,6 +410,10 @@ xi.mod =
     DARK_MAGIC_DURATION             = 1072, -- Increases Dark Magic spell durations by percentage (e.g. mod value 10 = +10% duration)
     ENHANCES_DARK_SEAL              = 1073, -- Enhances "Dark Seal" effect (Increases Dark Magic spell durations by 10% per Dark Seal merit while Dark Seal active)
 
+    -- Beastmaster
+    TANDEM_STRIKE                   = 271,  -- Grants a bonus to your and your pet's accuracy and magic accuracy when you and your pet are attacking the same target.
+    TANDEM_BLOW                     = 272,  -- Reduces amount of TP gained by enemies when striking them if you and your pet are attacking the same target.
```

Comment:
Following that logic, `TANDEM_BLOW_BONUS` would apply here.

And of course, same with the ones in modifier.h